tests/unit/backend/corpora/common/utils/test_json.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+import json
+import unittest
+from enum import Enum
+
+from sqlalchemy import Column, String
+
+from backend.corpora.common.corpora_orm import Base
+from backend.corpora.common.entities.entity import Entity
+from backend.corpora.common.utils.json import CustomJSONEncoder
+
+
+class DBTest(Base):
+    __tablename__ = "test"
+    id = Column(String, primary_key=True)
+    name = Column(String)
+
+
+class TestCustomJSONEncoder(unittest.TestCase):
+    def _test_json(self, test_value, expected_value):
```

Comment:
```suggestion
    def _verify_json_encoding(self, test_value, expected_value):
```
`test` made it seem like it was a unit test case.